src/roadmapper/milestone.py

```diff
@@ -59,6 +58,38 @@ def draw(self, painter: Painter) -> None:
                 self.diamond_height,
                 self.fill_colour,
             )
+
+        alignment = (
+            self.text_alignment.lower() if self.text_alignment is not None else None
+        )
```

Comment:
It would be nice if we caught wrong alignment cases already here. Then, we do not have to think of these wrong cases later on. This means you should move the `else` from below to this place.